pkg/sql/ir_generator.go

```diff
@@ -558,3 +563,16 @@ func parseShape(e *expr) ([]int, error) {
 	}
 	return shape, nil
 }
+
+func parseResultTable(intoStatement string) (string, error) {
+	resultTableParts := strings.Split(intoStatement, ".")
+	resultTable := ""
+	if len(resultTableParts) == 3 {
+		resultTable = strings.Join(resultTableParts[0:2], ".")
+	} else if len(resultTableParts) == 2 || len(resultTableParts) == 1 {
+		resultTable = intoStatement
+	} else {
+		return "", fmt.Errorf("error result table format, should be db.table.class_col or db.table or table")
+	}
+	return resultTable, nil
```

Comment:
How about making the condition more clearly?
``` go
    resultTableParts := strings.Split(intoStatement, ".")
    if len(resultTableParts) > 3 {
        return "", fmt.Errorf("bla")
    } else if len (resultTableParts) == 3 {
        return strings.Join(resultTableParts[0:2], "."), nill
    } else {
        return intoStatement, nil
    }
```

Reply:
FYI, package `tensorflow` has the same code snippet. If we were to change them, change both.
https://github.com/sql-machine-learning/sqlflow/blob/0b716a3b1bdcd038ddb401fe292a9169c6560e41/pkg/sql/codegen/tensorflow/codegen.go#L241-L249